### PR TITLE
DOCS-1339: PUBLISH CE 3.16.1

### PR DESCRIPTION
**Pull request by ctauchen**
Product Version(s):
CE 3.16.1 

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:
See #508 for approvals

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.16/releases.json [hunk 1]
```diff
@@ -13,243 +13,495 @@
...
+    },
+    "fluentd-windows": {
+      "image": "tigera/fluentd-windows",
+      "version": "v3.16.0"
+    },
+    "es-proxy": {
+      "image": "tigera/es-proxy",
+      "version": "v3.16.0"
+    },
+    "eck-kibana": {
```
Comment (bmckercher123) [inline calico-enterprise_versioned_docs/version-3.16/releases.json:326]: "7.17.9

File: calico-enterprise_versioned_docs/version-3.16/releases.json [hunk 2]
```diff
@@ -13,243 +13,495 @@
...
+      "version": "7.17.7"
+    },
+    "kibana": {
+      "image": "tigera/kibana",
+      "version": "v3.16.0"
+    },
+    "eck-elasticsearch": {
```
Comment (bmckercher123) [inline calico-enterprise_versioned_docs/version-3.16/releases.json:333]: "7.17.9

**Review by bmckercher123 — APPROVED** [review]
LGTM!

**Comment by doucol** [comment]
@sujeet-kr Are you going to merge this, or are you waiting on something?

**Comment by sujeet-kr** [comment]
Will merge in another 30 mins. Checking a final item.

File: calico-enterprise_versioned_docs/version-3.16/releases.json [hunk 3]
```diff
@@ -13,243 +13,495 @@
...
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "deep-packet-inspection": {
         "image": "tigera/deep-packet-inspection",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "eck-elasticsearch-operator": {
         "version": "2.5.0"
```
Comment (hjiawei) [inline calico-enterprise_versioned_docs/version-3.16/releases.json]: Is this updated to v2.6.1 for CE v3.16.1?

Comment (sujeet-kr) [reply calico-enterprise_versioned_docs/version-3.16/releases.json]: @rene-dekker can you confirm this please?

Comment (rene-dekker) [reply calico-enterprise_versioned_docs/version-3.16/releases.json]: Jiawei is correct, I pushed a commit.

**Review by sujeet-kr — APPROVED** [review]

**Pull request merged**